zingo-testutils/src/assertions.rs

```diff
@@ -33,3 +32,31 @@ pub async fn assert_send_outputs_match_client<NoteId>(
         );
     }
 }
+
+/// currently only checks if the fee matches
+/// this currently fails for any broadcast but not confirmed transaction: it seems like get_transaction_fee does not recognize pending spends
```

Comment:
Let's always use "pending" when we want an adjective that modifies a transaction, and reserve the word "broadcast" to be used as a verb, e.g. in the name of the function "propose_and_broadcast".